test/webrtc/ZombieRTCPeerConnection_spec.js

```diff
@@ -536,12 +536,43 @@ describe("A ZombieRTCPeerConnection", function () {
 		});
 	});
 
+	describe("removing a stream", function () {
+		describe("with an onremovestream handler", function () {
+			var onremovestream = Sinon.spy();
+
+			before(function (done) {
+				connection.onremovestream = onremovestream;
+				connection.removeStream("stream");
+				expect(onremovestream.called, "synchronous").to.be.false;
+				process.nextTick(done);
+			});
+
+			after(function () {
+				delete connection.onremovestream;
+			});
+
+			// The 'onremovestream' event refers to removing a remote stream.
+			it("does not invoke the onremovestream handler", function () {
+				expect(onremovestream.called).to.be.false;
```

Comment:
Should you also assert the lengths of ```this._localStreams``` in both of these tests and also check that the item is actually removed, I think.